docs/env_vars.md

```diff
@@ -130,20 +131,16 @@ Variable | Possible Value(s) | Description
 ---|---|---
 TORCHWEIGHTS     | [1] | use torch to initialize weights
 DISABLE_BACKWARD | [1] | don't do backward pass
+DIST             | [1] | enable distributed training
+STEPS            | [#] | number of steps
+FAKEDATA         | [1] | enable to use random data
 
 ### examples/benchmark_train_efficientnet.py & examples/hlb_cifar10.py
 
 Variable | Possible Value(s) | Description
 ---|---|---
 ADAM | [1] | use the Adam optimizer
 
-### examples/hlb_cifar10.py & xamples/hlb_cifar10_torch.py
-
-Variable | Possible Value(s) | Description
----|---|---
-STEPS    | [0-10] | number of steps
-FAKEDATA | [1]    | enable to use random data
```

Comment:
The FAKEDATA is still there but not working properly, should I fix it or remove it?

Reply:
can just remove it